Cargo.toml

```diff
@@ -1,8 +1,11 @@
 [package]
 name = "esp-exception-decoder"
```

Comment:
Should we rename the package to `esp-stacktrace-decoder` since  it's the name used everywhere?